javaScript/conditionals.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Javascript Task</title>
+    <script src="conditionals.js"></script>   
```

Comment:
Your script should be placed after the closing of the body tag and not in the head tag.